### PR TITLE
Add DistSQL for global clock

### PR DESCRIPTION
**Pull request by Pace2Car**
For #25222.

Changes proposed in this pull request:
  - Add DistSQL for global clock
  - Add related unit tests

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: kernel/global-clock/distsql/handler/src/main/java/org/apache/shardingsphere/globalclock/distsql/handler/update/AlterGlobalClockRuleStatementUpdater.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ */
+
+package org.apache.shardingsphere.globalclock.distsql.handler.update;
+
+import org.apache.shardingsphere.distsql.handler.ral.update.GlobalRuleRALUpdater;
+import org.apache.shardingsphere.globalclock.api.config.GlobalClockRuleConfiguration;
+import org.apache.shardingsphere.globalclock.distsql.parser.statement.updatable.AlterGlobalClockRuleStatement;
+
+/**
+ * Alter global clock rule statement handler.
```
Comment (RaigorJiang) [inline kernel/global-clock/distsql/handler/src/main/java/org/apache/shardingsphere/globalclock/distsql/handler/update/AlterGlobalClockRuleStatementUpdater.java:25]: Class name is `updater` but comment `handler`.

File: kernel/global-clock/distsql/parser/src/main/antlr4/imports/globalclock/RALStatement.g4 [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+propertiesDefinition
+    : PROPERTIES LP_ properties? RP_
+    ;
+
+properties
+    : property (COMMA_ property)*
+    ;
+
+property
+    : key=STRING_ EQ_ value=STRING_
```
Comment (RaigorJiang) [inline kernel/global-clock/distsql/parser/src/main/antlr4/imports/globalclock/RALStatement.g4:67]: The value of property should be typed `literal`

Please search for reference:
```
property
    : key=STRING_ EQ_ value=literal
```

Comment (Pace2Car) [reply kernel/global-clock/distsql/parser/src/main/antlr4/imports/globalclock/RALStatement.g4]: Fixed, THX

File: kernel/global-clock/distsql/handler/src/main/java/org/apache/shardingsphere/globalclock/distsql/handler/update/AlterGlobalClockRuleStatementUpdater.java (hunk 1 above)
Comment (Pace2Car) [reply kernel/global-clock/distsql/handler/src/main/java/org/apache/shardingsphere/globalclock/distsql/handler/update/AlterGlobalClockRuleStatementUpdater.java]: Fixed THX

**Review by RaigorJiang — APPROVED** [review]

**Pull request merged**